sass/layout/_headers.scss

```diff
@@ -18,10 +18,17 @@
         display: flex;
         flex: 1 1 0;
         min-width: 0;
+        padding: 2px 4px;
+        @include border-radius(4px);
 
         &:hover {
-            .header-dropdown__icon {
-                opacity: 1;
+            background: rgba(v(center-channel-color-rgb), 0.08);
+        }
+
+        &.active {
+            background: rgba(v(button-bg-rgb), 0.08);
+            .heading, .header-dropdown__icon {
+                color: v(button-bg);
             }
         }
 
```

Comment:
Can we update the following as well? Just to make sure spacing is correct within the header.
```
channel-header .flex-parent {
    padding: 0 16px 0 8px;
}
```

```
.channel-header__info {
    margin-top: 8px;
}